Utils/Elasticsearch/mapping/prodsys.mapping

```diff
@@ -145,11 +145,11 @@
           "type" : "keyword",
           "ignore_above" : 256
         },
-        "input_datasets" : {
+        "input" : {
           "type" : "keyword",
           "ignore_above" : 256
         },
-        "output_datasets" : {
+        "output" : {
```

Comment:
output_datasets -> output in mapping, but not in any place in code

Reply:
Right, that change was "hidden".
Here: https://github.com/PanDAWMS/dkb/pull/54/commits/5f5a1932edb474431d0703b1ac8b18cd63b20741 (DF/ora2es/oc: update ES mapping according to 5ea3d9d)

First there were implicitly specified field names for "input_datasets" and "output_datasets". But they had to be hardcoded into Python code, as there were no formal rule to get these names from original data. So they were shortened to "input" and "output" -- values of Oracle `dataset_type` field.